testsuite/shared/src/main/java/org/jboss/as/test/integration/management/cli/CliProcessWrapper.java

```diff
@@ -276,17 +276,21 @@ private boolean waitForClose() throws IOException {
     }
 
     private boolean outputHasPrompt(String prompt){
-        if(cliOutputBuffer.toString().contains("\n")) {
+        String buffer = cliOutputBuffer.toString();
+        if (buffer.startsWith("Exception")){
```

Comment:
Is there a reason for this change?

Reply:
yeah, it is here to aid with exception handling. 
With this change you will get proper exception reported in test result if exception occurs inside CLI execution.
Without this, it is swallowing the originating exception that happens inside CLI command.

Reply:
@jfdenise I'm going to merge this PR, but if you have any objection to this change, please let Tomaz know.